spec/models/instance_spec.rb

```diff
@@ -3,9 +3,9 @@
 require 'rails_helper'
 
 RSpec.describe Instance do
-  describe 'Scopes' do
-    before { described_class.refresh }
+  before { described_class.refresh }
 
+  describe 'Scopes' do
```

Comment:
I could revert this, but it's handy if we add other tests here in the future. 🤷🏻‍♀️